Mortgage-loan-calculator/src/app/admin-login/services/user.service.ts

```diff
@@ -1,16 +0,0 @@
-import { Injectable } from '@angular/core';
-import {HttpClient} from "@angular/common/http";
-import {Observable} from "rxjs";
-
-const API_URL = 'https://mortgage-loan-calculator-front-end2.onrender.com/auth/';
-@Injectable({
-  providedIn: 'root'
-})
-export class UserService {
-  constructor(private http: HttpClient) {
-  }
-
-  getPublicContent(): Observable<any> {
```

Comment:
Avoid usage of `any`. Set `tsconfig` to strict checks (including angular compiler).